### DISCUSSION TITLE
MOOSE users at UK universities?

### DISCUSSION THREAD

**Discussion by EdSheltonUKAEA** [idaholab/moose#18827] [Q&A General] [answered]
First, apologies if it isn't appropriate to ask this question in the discussion forum, and questions should be restricted only to technical matters about MOOSE.

I and colleagues at UKAEA would be interested to know of any researchers or groups currently using MOOSE at UK universities.  If there are any, it would be good to make contact to discuss possible collaboration and maybe support for PhDs.

If you'd like to get in touch, please reply to this post in the forum or send me a direct message.

**Comment by ngrilli** [comment, marked as answer]
Dear @EdSheltonUKAEA 

Here is one who just landed in the UK:
https://research-information.bris.ac.uk/en/persons/nicol%C3%B2-grilli

Best Regards,
Nicolò Grilli

**Reply by ngrilli** [reply]
Dear @EdSheltonUKAEA 

Sorry I have not been in touch, 
I don't know how to message you directly here and could not find your email online.

If you have still interest in co-supervision of PhD students, especially for the next call of CDT studentships, 
you can contact me to discuss at:
nicolo.grilli@bristol.ac.uk

Best Regards,
Nicolò

**Comment by GiudGiud** [comment]
@EdSheltonUKAEA this question is appropriate in the forum
@friedmud do you know anyone else in the UK using MOOSE?

**Reply by friedmud** [reply]
I don't, unfortunately.
